charts/hermes-relayer/templates/clientUpdateCronJob.yaml

```diff
@@ -10,6 +10,7 @@ spec:
     metadata:
       name: {{ include "hermes-relayer.fullname" . }}-job
     spec:
+      backoffLimit: {{ .Values.retryAmount }}
```

Comment:
Maybe we can use  `.Values.cronJob.backoffLimit` to put it in the update job context?